### PR TITLE
STDTC-11: Release v2.0.0

### PR DESCRIPTION
**Pull request by Vladyslav-Velytskyi**
## Purpose
Release module for the FOLIO 2020Q2. Refs [STDTC-11](https://issues.folio.org/browse/STDTC-11).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Change history for stripes-data-transfer-components
 
-## 1.1.0 (IN PROGRESS)
+## 2.1.0 (IN PROGRESS)
+
+## [2.0.0](https://github.com/folio-org/stripes-data-transfer-components/tree/v2.0.0) (2020-12-06)
```
Comment (YevheniiMaltsev) [inline CHANGELOG.md:5]: Please, add link to the full change log:
`[Full Changelog](https://github.com/folio-org/stripes-data-transfer-components/tree/v1.0.1...v2.0.0)`

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
...
 * Implement `SearchForm` component. UIDEXP-51.
 * Add `ProfilesLabel` and `SettingsLabel` components for second settings pane. UIDEXP-39.
 * Add mapping profiles pane component to display static data. UIDEXP-41.
```
Comment (YevheniiMaltsev) [inline CHANGELOG.md:8]: Please add the comment
`* Update translations`

File: CHANGELOG.md (hunk 1 above)
Comment (YevheniiMaltsev) [inline CHANGELOG.md:5]:
```suggestion
## [2.0.0](https://github.com/folio-org/stripes-data-transfer-components/tree/v2.0.0) (2020-06-12)
```

Comment (VictorSoroka1) [reply CHANGELOG.md]: For my PR I really watched the format you have used in ui-search. Always messing with this 😀.

**Pull request merged**